### DISCUSSION TITLE
Hub.docker.com: cleaning up old docker images ?

### DISCUSSION THREAD

**Discussion by barthanssens** [eclipse-rdf4j/rdf4j#5089] [General]
Even though there is no hard limit (AFAIK) on the amount of disk space we can use on hub.docker.com, I think it would be nice to clean up older / lesser unused images.

So I was thinking of removing the 4.3.2 / 4.3.3 / 4.3.4 / 4.3.5 / 4.3.6 images from hub.docker.com, unless there are objections ?

(this would still leave the even older versions 3.7.7, 4.1.3, 4.2.3 on hub.docker, and of course 4.3.7 /...-13 + 5 .0.x will also remain available)

(and it has no impact whatsoever on the SDK/one-jar downloads from the Eclipse website and mirrors)

**Comment by hmottestad** [comment]
Last time when we cleaned up old versions we got some complaints from people that were locked into an older version. Don't know if the same would happen now, but it's possible someone has a hard dependency on a specific patch release because of some bug that they haven't told us about or we haven't fixed. 

Can we instead disable some images and then delete them in 6 months if we don't hear anything?

**Reply by barthanssens** [reply]
Hmz ok, nevermind then.

I don't think one can disable / hide images, the web interface is just about pushing or deleting images...

Also, the docker interface does indicate that the 4.2 and 4.3.x images are still being pulled (at exactly the same date, so I'm guessing someone is mirroring all these images), but not _how many_ pulls per image / tag
